feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -90,14 +98,14 @@ func configureDUT(t *testing.T, dut *ondatra.DUTDevice, p1 *ondatra.Port, p2 *on
 	d := gnmi.OC()
 
 	// Configure ingress interface
-	t.Logf("*** Configuring interfaces on DUT ...")
+	t.Logf("*** Configuring Interfaces on DUT ...")
```

Comment:
revert line, no need to capitalize "Interfaces"

Reply:
done in new commit.